### ISSUE TITLE
Feature request: Public profile pages

### ISSUE THREAD

**Issue by mrbiber** [zenodo/zenodo#911]
I'd like to have a link on my website that links to all my publications on Zenodo. This seems not be possible at the moment.

**Comment by christofs** [comment]
I'm using the URL generated when searching for my name as a workaround: https://zenodo.org/search?page=1&size=100&q=%22Sch%C3%B6ch,%20Christof%22

**Comment by ccamara** [comment]
+1 to this feature request.

IMHO having a public profile is a must if scholars are to use zenodo as a page for promoting their work or to be updated about other scholars' + it would help to find content by the same author (provided that author's name had a link to their public profile).

Using the search's URL is a workaround which unfortunately is useless if you have a popular name or if your name has non-english characters

**Comment by mohtamohit** [comment]
Hello everyone, I would be glad to work on this. I've a good experience in full stack web development and I would like to work on this under the umbrella of GSoC 17. Is it okay if I start working on this after I'm done with submitting my proposal on the GSoC website? 
Meanwhile, you may review my profile at https://docs.google.com/document/d/1QwQP37iEl2qfiQ6qkx2Pzheg3IQXpXK6IercuSKAgJs/edit?usp=sharing

**Comment by KyleMoJo** [comment]
+ 1 indeed, this would add enormous value to the platform!

**Comment by ccamara** [comment]
@mohtamohit according to[ contributing guidelines](https://github.com/zenodo/zenodo/blob/master/CONTRIBUTING.rst) you can do so! Please read this developers' guidelines: https://zenodo.readthedocs.io/en/latest/

**Comment by krzysztof** [comment]
The feature is already developed, and we hope to merge it soon: https://github.com/zenodo/zenodo/pull/1249

**Comment by ccamara** [comment]
Great news! Glad to hear that.

Carlos Cámara
http://carloscamara.es

2018-01-22 10:08 GMT+01:00 Krzysztof Nowak <notifications@github.com>:

**Comment by ccamara** [comment]
@krzysztof  do you have any progress update about this feature? If I'm not wrong, the latest pull request activity dates from august 2017, which is pending review.

**Comment by ccamara** [comment]
I see that #1249 PR has been merged, but I cannot find the public profile page anywhere and this issue is still open, so I am assuming that the feature is not yet implemented. Could you please update about it?

**Comment by yaxu** [comment]
Yes it'd be great to have this! I'm trying to work around it by creating a community with my name, but it's tedious to add publications one-by-one, and creating a community just for myself seems a little narcissistic..

**Comment by yaxu** [comment]
The above-mentioned workaround doesn't really work, as you can't add a co-authored paper that someone else has uploaded (without bothering them to do it for each one, or duplicating the upload..)

**Comment by pasabanov** [comment]
Any update on this feature?
